ProxyAppsSuite/tests/examini.yaml

```diff
@@ -0,0 +1,103 @@
+
+examini:
+   summary: Research vehicle for particule codes in Molecular Dynamics which allows modular independent investigation of different aspects such as force calculation, communication, neighbor list construction and binning.
+  
+
+   build:
+       modules: [ gcc, cmake, openmpi/2.1.2 ]
+       source_path: ExaMini
```

Comment:
```suggestion

```

I kept getting an error because `test_src/ExaMini` did not exist for me. I deleted this line and it seemed to work. 

Two suggestions: 
1. delete this line -- you don't really need it since you `git clone` the repo in line 24
2. `source_url: https://github.com/ECP-copa/ExaMiniMD/archive/master.zip` so that Pavilion will download this in `test_src` (you won't need to `git clone` at the bottom, but I think you'd have to edit the lines below that a little).